File: services/22-rails/app/controllers/health_controller.rb
class HealthController < ActionController::API
  def hello
    render json: { message: 'Hello from Rails 8.0', framework: '22-rails', version: '1.0.0' }
  end

  def health
    render json: { status: 'ok', version: '1.0.0' }
  end

  def liveness
    render json: { status: 'ok' }
  end

  def readiness
    render json: { status: 'ok' }
  end
end

File: services/25-play/app/controllers/HealthController.scala
package controllers

import play.api.mvc._
import play.api.libs.json._
import javax.inject._

@Singleton
class HealthController @Inject()(val controllerComponents: ControllerComponents) extends BaseController {

  def hello: Action[AnyContent] = Action {
    Ok(Json.obj("message" -> s"Hello from Play 3.0", "framework" -> "25-play", "version" -> "1.0.0"))
  }

  def health: Action[AnyContent] = Action {
    Ok(Json.obj("status" -> "ok", "version" -> "1.0.0"))
  }

  def liveness: Action[AnyContent] = Action { Ok(Json.obj("status" -> "ok")) }
  def readiness: Action[AnyContent] = Action { Ok(Json.obj("status" -> "ok")) }
}
